prow/jobs/kyma/components/kyma-components-static-checks.yaml

```diff
@@ -285,7 +285,7 @@ presubmits: # runs on PRs
         prow.k8s.io/pubsub.project: "sap-kyma-prow"
         prow.k8s.io/pubsub.runID: "pull-event-publisher-proxy-unit-test"
         prow.k8s.io/pubsub.topic: "prowjobs"
-      run_if_changed: '^components/event-publisher-proxy/.*\.go$'
+      run_if_changed: '^components/event-publisher-proxy/(.*\.go$|go\.mod$|go\.sum$)'
```

Comment:
what about yaml's? Shouldn't we also run tests incase our CRD changes, for example?

Reply:
I changed it to run whenever we change anything that could affect the outcome of running the unit-tests.

Reply:
Because golang does not support regex lookaround, I changed it to watch for all changes as agreed.